libs/importer/src/importers/bitwarden/bitwarden-json-importer.ts

```diff
@@ -47,12 +47,7 @@ export class BitwardenJsonImporter extends BaseImporter implements Importer {
   }
 
   async parse(data: string): Promise<ImportResult> {
-    // parsing ssh keys needs an sdk initialized
-    // remove this once the sdk gets initialized automatically
-    const isRunningInJest = typeof jest !== "undefined";
-    if (!isRunningInJest) {
-      await firstValueFrom(this.sdkService.client$);
-    }
+    await firstValueFrom(this.sdkService.client$);
```

Comment:
⛏️ Seems like you went too far with this change. The comment should still be included, correct? If so, make sure you call out the tech debt with a FIXME.

```suggestion
    // initializes the SDK
    // FIXME: remove this once the SDK gets initialized automatically
    await firstValueFrom(this.sdkService.client$);
```

🤔 If this line is no longer needed for initialization, what does it do? Why isn't the value returned by `client$` used?

Reply:
> 🤔 If this line is no longer needed for initialization, what does it do? Why isn't the value returned by client$ used?

Yeah, I agree this comment makes sense. It is unfortunate that this needs to be done using such a confusing line of code at the moment.